frontend/src/components/dashboard/ArtworkCloseUpCard.js

```diff
@@ -50,10 +50,15 @@ export default function ArtworkCloseUpCard() {
       'consequat tellus, et venenatis risus quam ' +
       'nec massa.';
 
+  const ttsLink = 'https://storage.cloud.google.com/tts-audio/picture-id';
+
   useEffect(() => {
     const params = new URLSearchParams();
     params.append('text', description);
-    fetch('/api/v1/tts', {method: 'POST', body: params});
+    params.append('id', 'picture-id');
+    fetch('/api/v1/tts', {method: 'POST', body: params})
+        .then(() => document.getElementById('audio')
+            .setAttribute('src', ttsLink));
```

Comment:
Just wondering, why do you set the link after you fetch as opposed to just setting it manually in the code?  Is it to ensure that the audio file was uploaded before allowing the user to play the audio?

Reply:
yes, that's it. The link isn't valid until the file's been uploaded. I need to add some error catching which I will do in another PR.